ligo/stablecoin/contract-metadata.ligo

```diff
@@ -0,0 +1,21 @@
+// SPDX-FileCopyrightText: 2020 TQ Tezos
+// SPDX-License-Identifier: MIT
+
+#include "spec.ligo"
```

Comment:
1. Are we using anything from `spec.ligo`?
2. Maybe let's call this module `tzip16-metadata`? Now we have `metadata.tz` and `contract-metadata.tz`, it's not obvious what the difference between them is.

Reply:
Removed spec.ligo import and renamed the contract.